artifacts/deploy/karmada-controller-manager.yaml

```diff
@@ -32,6 +32,7 @@ spec:
             - --failover-eviction-timeout=30s
             - --controllers=*,hpaReplicasSyncer
             - --feature-gates=PropagationPolicyPreemption=true
+            - --skipped-propagating-namespaces=kube-.*
```

Comment:
Why bring this flag here?

Reply:
The default value of flag ```--skipped-propagating-namespaces``` is ```kube-.*```, and I found one another installation method also add this flag, like 
https://github.com/karmada-io/karmada/blob/bb1d2e8f6feaf8881931129ef4abf8e573a9ccd4/cmd/controller-manager/app/options/options.go#L194-L196

Reply:
Thanks for correcting me, I understood it wrong before. This file actually adds a default value to the controller-manager's flag ```--skipped-propagating-namespaces```, so it's no need to bring this flag here again.